### PR TITLE
changed schemas to reflect scanTime change instead of lastScanned

### PR DESCRIPTION
**Pull request by fakebrandonchan**
dependent on https://github.com/asecurityteam/nexpose-asset-producer/pull/36/files

### CODE REVIEW

**Review by mikerott — CHANGES_REQUESTED** [review]
Similar to the comment I made in https://github.com/asecurityteam/nexpose-asset-producer/pull/36/files -- can we preserve the `lastScanned` property and add the `scanTime` then use whichever one is non-zero value.  This makes the change backward compatible and safe during the corresponding change of the schema in the other services.

File: gateway-incoming.yaml [hunk 1]
```diff
@@ -97,7 +97,6 @@ components:
       required:
         - id
         - ip
-        - lastScanned
```
Comment (mikerott) [inline gateway-incoming.yaml:100]: task:  should `scanTime` be among the required?

File: gateway-incoming.yaml [hunk 2]
```diff
@@ -122,7 +125,6 @@ components:
       required:
         - id
         - hostname
-        - lastScanned
```
Comment (mikerott) [inline gateway-incoming.yaml:125]: task:  should `scanTime` be among the required?

File: gateway-outgoing.yaml [hunk 3]
```diff
@@ -100,7 +100,6 @@ components:
       required:
         - id
         - ip
-        - lastScanned
```
Comment (mikerott) [inline gateway-outgoing.yaml:103]: task:  should `scanTime` be among the required?

**Review by mikerott — APPROVED** [review]
Task:  I expected to see code changes that took the inbound `scanTime` and set it on the outbound `lastScanned` and the inverse when the prior service only sends one of the two values.  I don't see any code like that in the PR.  I'm approving this with much fear.

**Review by faldridge — APPROVED** [review]

**Pull request merged**